igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import _thread
+import argparse
+import subprocess
+
+from time import time, sleep
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Ping colecctor script for Graphana ')
+    parser.add_argument('adress', metavar='URL', type=str, nargs='+',
+                        help='the address zu ping to')
+    parser.add_argument('-c', '--count', type=int, default=20,
+                        help='how many tims should the script tries to ping')
+    parser.add_argument('--prefix', default='ping',
+                        help='the path to the value in Graphana')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    hostname = args.adress
+    count = args.count
+    prefix = args.prefix
+
+    i = 1
+    hosts = ''
+    for host in hostname:
```

Comment:
instead of tracking `i` separately just do `for i, host in enumerate(hostname)`. it will start at zero though. keep that in mind

Reply:
`enumerate` also takes a starting number, so it could be: `enumerate(hostname, 1)`
However, `i` is never changed inside the for-loop. Is it really needed?

Reply:
compleatly changed